types/react-router-redux/index.d.ts

```diff
@@ -31,7 +31,7 @@ export interface RouterState {
     location: Location | null;
 }
 
-export function routerReducer(state?: RouterState, action?: RouterAction): RouterState;
+export function routerReducer(state?: RouterState, action?: RouterAction | AnyAction): RouterState;
```

Comment:
Does `action?: AnyAction` suffice instead of `action?: RouterAction | AnyAction`?